run.sh

```diff
@@ -279,6 +279,18 @@ if [[ "$NODRIVER" != 1 ]]; then
         set -- --cpus 1 --sysmem 256 "$@"
         set -- --omps "$NUM_OMPS" --ompthreads "$THREADS_PER_OMP" "$@"
         set -- --numamem "$RAM_PER_OMP" "$@"
+        if [[ "$USE_CUDA" == 1 ]]; then
```

Comment:
I think that it would be better to have a separate case of computing threads etc for the case where we have `USE_OPENMP=1` and `USE_CUDA=1`. What i'm doing right now is also ensuring that we have a reserved thread per GPU as well. Maybe we need to do more special things to make this configuration work out, so having a separate case makes sense.

Reply:
I agree that it makes sense to reserve more cores depending on the number of GPUs. However, I think we should also be doing that in the `USE_CUDA=1 USE_OPENMP=0` case. Added this with e502a72fccba8691a275c7fdb406be3858d5ced7.

For now the logic between `USE_CUDA=1 USE_OPENMP=1` and `USE_CUDA=0 USE_OPENMP=1` cases is uniform, so I'd prefer to keep it grouped as-is to avoid duplication, and we can revisit later if we need to diverge.